Source/MonoGame.Extended.Tests/AngleTest.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using Microsoft.Xna.Framework;
+using NUnit.Framework;
+
+namespace MonoGame.Extended.Tests
+{
+    [TestFixture]
+    public class AngleTest
+    {
+        private const float DELTA = 0.0000001f;
```

Comment:
Correct.

You could read through the Microsoft guidelines if you like, but as a rule of thumb the easiest thing to do is to look at examples in the .NET framework. For example, one of the constants in the .NET framework is [`int.MinValue`](https://msdn.microsoft.com/en-us/library/system.int32.minvalue(v=vs.110).aspx).

`private` variable naming conventions are not actually defined in the guidelines but as you've already noticed we're using the underscore prefix. The reason for this is to differentiate private member variables from properties when you're reading the code. It's also helpful because you don't have to qualify everything with `this`.

I use a tool called [Reshaper](https://www.jetbrains.com/resharper/) that really helps with this kind of stuff. It does cost a bit of money but even using the 30 day trial is useful to learn loads of stuff about how to write really clean code. It kind of teaches you as you go.

Btw.. I've noticed a huge improvement in your PR's lately. I appreciate the effort.